packages/sanity/src/core/config/configPropertyReducers.ts

```diff
@@ -308,6 +309,31 @@ export const documentCommentsEnabledReducer = (opts: {
   return result
 }
 
+export const internalTasksReducer = (opts: {
+  config: PluginOptions
+}): {footerAction: ReactNode} | undefined => {
+  const {config} = opts
+  const flattenedConfig = flattenConfig(config, [])
+
+  const result = flattenedConfig.reduce(
+    (acc: {footerAction: React.ReactNode} | undefined, {config: innerConfig}) => {
```

Comment:
Should this be `ReactNode`, rather than `React.ReactNode`? It's explicitly imported like this: `import {type ReactNode} from 'react'`.